pages/mentor/04-defining_a_project__ideas_list.md

```diff
@@ -46,4 +46,4 @@ The project you propose will define the tone and scope of your organization's pa
 
 **Pro Tip**: Maintain an "ideas page" with a running list of entry projects year-round. This can benefit your development effort throughout the year. It can also make your organization's GSoC application easier to put together the following summer.
 
-**Don't Be That Guy**: Don't propose projects that neither you nor anyone else wants to mentor.
+**Don't Be That Person**: Don't propose projects that neither you nor anyone else wants to mentor.
```

Comment:
Maybe we can have all of these changes in a single PR (either this or https://github.com/google/gsocguides/pull/20?)

Reply:
I'm editing directly via github's UI, so make its a bit challenging :(